### PR TITLE
Adding CfP link to nev bar

### PR DESCRIPTION
**Pull request by Cheukting**

### CODE REVIEW

File: hugo.toml [hunk 1]
```diff
@@ -14,6 +14,11 @@ name = 'About'
 pageRef = '/about'
 weight = 20
 
+[[menus.main]]
+name = 'CfP'
+pageRef = 'https://pretalx.com/pyladiescon-2024/cfp'
```
Comment (Mariatta) [inline hugo.toml:19]: The link doesn't work on the preview site 😕

File: hugo.toml [hunk 2]
```diff
@@ -14,6 +14,11 @@ name = 'About'
 pageRef = '/about'
 weight = 20
 
+[[menus.main]]
+name = 'CfP'
```
Comment (Mariatta) [inline hugo.toml:18]: How about "Speaking" instead of CFP

File: hugo.toml (hunk 1 above)
Comment (Cheukting) [reply hugo.toml:19]: I think we may have to hard code it (like #contact)

File: hugo.toml (hunk 2 above)
Comment (Cheukting) [reply hugo.toml:18]: Sounds good

**Comment by Cheukting** [comment]
Ready when you are @Mariatta

**Review by Mariatta — APPROVED** [review]
Thanks!

**Pull request merged**